### ISSUE TITLE
After updated to BiglyBT to 3.6.0.0, it keep looking for update on 3.5.0.0

### ISSUE THREAD

**Issue by SUCM** [BiglySoftware/BiglyBT#3233]
as title stated. I had tried reboot and restart BiglyBT several times already. every time I start BiglyBT, it runs an update task for 3.5.0.0. I wait until it finishes and clear&remove it in the progress window, but the next start runs a new one again. see OS info and update log below
```

Java 11.0.16.1 (64 bit)
  Oracle Corporation
c:\program files\java\jdk-11.0.16.1

SWT v4942r22, win32, zoom=100, dpi=96
Windows 10 v10.0, amd64 (64 bit)
B3.6.0.0/4 az3 en_US_classic
```

```
Started 4/27/24, 12:48 AM
Checking for Updates - added
Checking for Updates
    Core Patch Checker -     Adding update: Core Patch Checker
    BiglyBT Core -     Core: latest_version = '3.5.0.0', file = 'http://version.biglybt.com/biglybt-files/core/BiglyBT_Update_3.5.0.0_win64.zip.torrent'
    BiglyBT Core - Finished
    SWT library -     SWT: current version = 4942r22, latest version = 4942r22
    Core Patch Checker - Finished
    SWT library - Finished
    Mandatory plugins -     Loading details for azupdater/BiglyBT Update Support
    Platform-specific support -     Win32: current = 1.27, latest = 1.27
    Mandatory plugins -     Loading details for azupdater/BiglyBT Update Support
    Platform-specific support - Finished
    Non-mandatory plugins -     Plugin: azlocprovversions (current=0.6.4, latest=0.6.4)
    Non-mandatory plugins -     Plugin: azlocprovversions (current=0.6.4, latest=0.6.4)
.
.
.
```

**Comment by parg** [comment]
The installer on biglybt.com is still 3.5

**Comment by parg** [comment]
ah, ok, there was a glitch in the update matrix - should be fixed

**Comment by SUCM** [comment]
just restarted again, but it kicks off another one

```
Started 4/27/24, 2:11 AM
Checking for Updates - added
Checking for Updates
    Core Patch Checker -     Adding update: Core Patch Checker
    BiglyBT Core -     Core: latest_version = '3.5.0.0', file = 'http://version.biglybt.com/biglybt-files/core/BiglyBT_Update_3.5.0.0_win64.zip.torrent'
    BiglyBT Core - Finished
    Core Patch Checker - Finished
    SWT library -     SWT: current version = 4942r22, latest version = 4942r22
    SWT library - Finished
```

**Comment by SUCM** [comment]
I just downloaded Win X64 installer from https://www.biglybt.com/download/ and see it's 3.6.0.0, so both github and biglybt.com are at 3.6.0.0. not sure about stub installer since I'm not able to pick jre with the stub installer

**Comment by SUCM** [comment]
also, the 3.6.0.0 was from BiglyBT's own auto update. it created a 3.6.0.0 update torrent and downloaded 3.6.0.0 on it's own then prompted for a restart to update. after the restart, I'm in the current situation

**Comment by parg** [comment]
OK, but it isn't actually doing anything right? Just reporting 3.5. It is an artifact of the phased update currently in progress. I guess I can fix it so it says 3.6 consistently

**Comment by SUCM** [comment]
I see. it did performed a few plugin updates. hence I was worried that it would do something to the core component. if it's just displaying wrong version and won't actually downgrade then I'm ok to ignore it for a while
